trinity/utils/filesystem.py

```diff
@@ -0,0 +1,8 @@
+import os
+
+
+def ensure_path_exists(dir_path):
+    """
+    Make sure that a path exists
+    """
+    os.makedirs(dir_path, exist_ok=True)
```

Comment:
This seems so small that it's worth inlining. Then we can delete the tests, too.

Reply:
Yeah, this was a carry-over from python2 support that was necessary but is no longer.  👍